litellm/tests/test_verify_openai_key.py

```diff
@@ -0,0 +1,7 @@
+from litellm import verify_access_key
+
+def bad_key():
+    key = "bad-key"
+    response = verify_access_key(key)
+    print(f"response: {response}")
+bad_key()
```

Comment:
can you add a test for good key. you can use os.environ['OPENAI_API_KEY'] for the good key. Our CI/CD pipeline uses that too